lib/index.js

```diff
@@ -154,61 +154,79 @@ function saveScore() {
-  if (board.modes.vim === false ) {
-    $(document).off();
-    $(document).keydown(function(e) {
-      if ((e.keyCode === 37) && snake.direction !== "right") {
-        snake.direction = "left";
-      } else if ((e.keyCode === 38)&& snake.direction !== "down") {
-        snake.direction = "up"
-      } else if ((e.keyCode === 39) && snake.direction !== "left") {
-        snake.direction = "right"
-      } else if ((e.keyCode === 40) && snake.direction !== "up") {
-        snake.direction = "down"
-      }
-    })
-  }
-  if (board.modes.vim === true){
-    $(document).off();
-    $(document).keydown(function(e) {
-      if ((e.keyCode === 72) && snake.direction !== "right") {
-        snake.direction = "left";
-      } else if ((e.keyCode === 75)&& snake.direction !== "down") {
-        snake.direction = "up"
-      } else if ((e.keyCode === 76) && snake.direction !== "left") {
-        snake.direction = "right"
-      } else if ((e.keyCode === 74) && snake.direction !== "up") {
-        snake.direction = "down"
-      }
-    })
+  $(document).off();
+  $(document).keydown(function(e) {
+      changeDirection(e)
+  });
+}
+
+function changeDirection(e) {
+  if (shouldMoveLeft(e)) {
+    snake.direction = "left";
+  } else if (shouldMoveUp(e)) {
+    snake.direction = "up";
+  } else if (shouldMoveRight(e)) {
+    snake.direction = "right"
+  } else if (shouldMoveDown(e)) {
+    snake.direction = "down"
   }
 }
 
-function snakeCollidesWithFood() {
-  if (food.x < snake.x + snake.width &&
-     food.x + food.width > snake.x &&
-     food.y < snake.y + snake.height &&
-     food.height + food.y > snake.y) {
-     // collision detected!
-     food.eaten;
-     food = new Food(options);
-     board.checkLocation(portalSet, food, snake);
-     snake.addNode;
-     board.addPoints;
-     board.increaseLevel;
-     if (board.powerUpAble === true ) {
-       board.modes.powerup = true
-     }
-   }
-};
+function shouldMoveLeft(e) {
+  if (board.modes.vim === true ) {
+    return (e.keyCode === 72) && snake.direction !== "right"
+  } else {
+    return (e.keyCode === 37) && snake.direction !== "right"
+  }
+}
 
-function snakeCollidesWithPowerUp() {
-  if (powerup.x < snake.x + snake.width &&
-     powerup.x + powerup.width > snake.x &&
-     powerup.y < snake.y + snake.height &&
-     powerup.height + powerup.y > snake.y) {
-     // collision detected!
-     board.modes.powerup = false;
-     board.enablePowerUp
+function shouldMoveUp(e) {
+  if(board.modes.vim === true ) {
+    return (e.keyCode === 75) && snake.direction !== "down"
+  } else {
+    return (e.keyCode === 38) && snake.direction !== "down"
   }
-};
+}
+function shouldMoveDown(e) {
+  if(board.modes.vim === true ) {
+    return (e.keyCode === 74) && snake.direction !== "up"
+  } else {
+    return (e.keyCode === 40) && snake.direction !== "up"
+  }
+}
+function shouldMoveRight(e) {
+  if(board.modes.vim === true ) {
+    return (e.keyCode === 76) && snake.direction !== "left"
```

Comment:
You miiiiight want to consider breaking the back end of this && statement into it's own function. The logic being that if you chance 'left' to 'Left' for some reason, you have to change it in two places... 

LGTM though!